### ISSUE TITLE
new mode: 1vs1 of soup (the other went wrong ...)

### ISSUE THREAD

**Issue by Sh4dowForce14** [versai-network/Updates#7]
would be like the nodebuff mode, only instead of potion would be soup, this would help players who do not have much agility to throw a potion

**Comment by artulloss** [comment]
You want this as 1v1s only? I can do that pretty easily.

**Comment by Sh4dowForce14** [comment]
yes I just wanted this, I see it in the computer Minecraft and I think it's very cool :) wanted to play this on the cell phone could bring this in a future update?

**Comment by artulloss** [comment]
Yeah I can it's all configurable so I would just need to add in a kit and make soup give health

**Comment by Sh4dowForce14** [comment]
ok thank you so much for adding this mode to the future and just one more thing, could you leave the server to 1.11 until 1.12? I and several more friends of mine are not able to play, because when we enter a duel leaves the server and we are accustomed to playing at 1.11 would be very happy :)

**Comment by artulloss** [comment]
Oh sorry lol

**Comment by Sh4dowForce14** [comment]
ok :)
